sklearn/linear_model/randomized_l1.py

```diff
@@ -380,8 +383,12 @@ class RandomizedLogisticRegression(BaseRandomizedLinearModel):
 
     Parameters
     ----------
-    C : float, optional, default=1
+    C : float or array of shape [n_reg_parameter], optional, default=1
         The regularization parameter C in the LogisticRegression.
+        When C is an array, fit will take each regularization parameter in C 
+        one by one for LogisticRegression and store results for each one 
+        in all_scores_, where columns and rows represent corresponding 
```

Comment:
can you please use double backticks around ``all_scores_``? Not only is it a variable, the documentation generation breaks if something ends with an underscore and is not escaped.